libbeat/publisher/queue/spool/spool_test.go

```diff
@@ -44,9 +45,12 @@ type testLogger struct {
 	t *testing.T
 }
 
+type silentLogger struct{}
+
 func init() {
 	flag.Int64Var(&seed, "seed", time.Now().UnixNano(), "test random seed")
 	flag.BoolVar(&debug, "noisy", false, "print test logs to console")
+	flag.BoolVar(&verboseLogger, "verboseLogger", false, "use verbose logger")
```

Comment:
Is it unused? And shouldn't it be enough with the `noisy` flag?

Reply:
The flag `noisy` (debug) means that you want to log more data than just single log line:

```
func (l *testLogger) reportf(level string, str string, vs []interface{}) {
	str = level + ": " + str
	l.t.Logf(str, vs...)
	if debug {
		fmt.Printf(str, vs...)
	}
}
```
Leaving it as is prints thousands of log lines and exceeds Travis limit, hence the silent logger.

Reply:
You're right, I will use a single flag for debug/noisy logger.